snap/snapcraft.yaml

```
@@ -21,15 +21,17 @@ parts:
   connectedhomeip:
     plugin: nil
     build-environment:
-      - TAG: master
+      - UPSTREAM_TAG: master
+      - PACKAGE_VERSION: snap
```

Package version is the snap version, since this is a snap package.

Maybe:
```suggestion
      - VERSION_METADATA: snap
```